Python/specialize.c

```diff
@@ -1917,36 +1869,25 @@ specialize_py_call(PyFunctionObject *func, _Py_CODEUNIT *instr, int nargs,
         SPECIALIZATION_FAIL(CALL, SPEC_FAIL_CALL_PEP_523);
         return -1;
     }
-    if (kind != SIMPLE_FUNCTION) {
-        SPECIALIZATION_FAIL(CALL, kind);
+    int argcount = -1;
+    if (kind == SPEC_FAIL_CODE_NOT_OPTIMIZED) {
+        SPECIALIZATION_FAIL(CALL, SPEC_FAIL_CODE_NOT_OPTIMIZED);
         return -1;
     }
```

Comment:
`CALL_PY_GENERAL` can handle this, right?

Reply:
Yes, but I don't want to hide these cases, otherwise we will never know if they are a problem. 

We could record stats for failed specialization that have a fall back, like this case, but I think that's a bit too intrusive for 3.13.